config/routes.rb

```diff
@@ -1,8 +1,12 @@
 Rails.application.routes.draw do
-  root to: "users#index"
+  root to: "dashboard#index"
 
   devise_for :users
   resources :users, except: [:destroy, :show]
 
-  resources :books, except: [:show]
+  resources :books, except: [:show] do
+    member do
```

Comment:
2 options:

1. `collection` -> `/books/toggle_featured/`
2. `member` -> `/books/:id/toggle_featured/`

So you use `member` because you are toggling a single resource.